src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectIT.java

```diff
@@ -1,385 +1,652 @@
+                + "timestampcol TIMESTAMP, " //
+                + "tinyinteger TINYINT, " //
+                + "varcharcol VARCHAR(10), " //
+                + "binarycol BINARY, " //
+                + "datecol DATE)");
+        statementHive.execute("TRUNCATE TABLE " + TABLE_HIVE_ALL_DATA_TYPES);
+        statementHive.execute("INSERT INTO " + TABLE_HIVE_ALL_DATA_TYPES
+                + "(arraycol,biginteger,boolcolumn,charcolumn,decimalcol,doublecol,floatcol,intcol,mapcol,smallinteger,"
+                + "stringcol,structcol,timestampcol,tinyinteger,varcharcol,binarycol,datecol) SELECT " //
+                + "ARRAY('etet','ettee'), " //
+                + "56, " //
+                + "true, " //
+                + "'2', " //
+                + "53, " //
+                + "56.3, " //
+                + "5.199999809265137, " //
+                + "85, " //
+                + "map('jkljj',5), " //
+                + "2, " //
+                + "'tshg', " //
+                + "named_struct('a',2,'b',4), " //
+                + "timestamp '2017-01-02 13:32:50.744', " //
+                + "1, " //
+                + "'tytu', " //
+                + "'MTAxMA==', " //
+                + "date '1970-01-01' " //
+                + "FROM " + TABLE_HIVE_SIMPLE);
+    }
+
+    private static void createJdbcConnection() throws SQLException {
+        integrationTestSetupManager.createConnection(statementExasol, CONNECTION_HIVE_JDBC,
+                "jdbc:hive2://" + DOCKER_IP_ADDRESS + ":" + HIVE_EXPOSED_PORT + "/" + SCHEMA_HIVE, //
+                HIVE_USERNAME, HIVE_PASSWORD);
+    }
+
+    private static void createVirtualSchema(final String virtualSchemaName, final Optional<String> additionalParameters)
+            throws SQLException {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("CREATE VIRTUAL SCHEMA ");
+        builder.append(virtualSchemaName);
+        builder.append(" USING " + SCHEMA_EXASOL + "." + ADAPTER_SCRIPT_EXASOL + " WITH ");
+        builder.append("SQL_DIALECT     = 'HIVE' ");
+        builder.append("CONNECTION_NAME = '" + CONNECTION_HIVE_JDBC + "' ");
+        builder.append("SCHEMA_NAME     = '" + SCHEMA_HIVE + "' ");
+        additionalParameters.ifPresent(builder::append);
+        final String sql = builder.toString();
+        LOGGER.info("Creating virtual schema with query: " + sql);
+        statementExasol.execute(sql);
     }
 
     @Test
-    void testLeftJoin() throws SQLException {
-        final String query = "SELECT * FROM  " + VIRTUAL_SCHEMA + ".t1 a LEFT OUTER JOIN  " + VIRTUAL_SCHEMA
-                + ".t2 b ON a.x=b.x ORDER BY a.x";
-        final ResultSet result = executeQuery(query);
-        assertAll(() -> assertNextRow(result, 1L, "aaa", null, null), () -> assertNextRow(result, 2L, "bbb", 2L, "bbb"),
-                () -> assertFalse(result.next()));
-    }
+    public void testSetup() throws SQLException {
```

Comment:
Why is this called `testSetup()`?
What does it really test?

Reply:
Probably that virtual schemas were created correctly. Also legacy

Reply:
fixed